src/lwc/geDonationMatchingOpportunityCard/geDonationMatchingOpportunityCard.js

```diff
@@ -124,7 +137,7 @@ export default class geDonationMatchingOpportunityCard extends LightningElement
     * @param {object} event: Custom Event object containing payment data.
     */
     handleUpdatePayment(event) {
-        dispatch(this, 'updateselecteddonation', event.detail);
```

Comment:
What's motivating the move from the shared dispatch from utilCommon to the dispatchEvent call here?